src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/88473ad0d5c44bf50e979695825f8c06779d3ac8